vyxal/LazyList.py

```diff
@@ -21,6 +21,17 @@ def wrapped(*args, **kwargs):
     return wrapped
 
 
+def infinite_lazylist(fn):
```

Comment:
Could we just add an `is_inf` optional parameter to the existing `lazylist` decorator?